data/remnant ships.txt

```diff
@@ -414,6 +414,66 @@ ship "Pelican"
 
 
 
+ship "Penguin"
+	sprite "ship/penguin/penguin"
+		"frame rate" 3
+	thumbnail "thumbnail/penguin"
+	attributes
+		category Transport
+		cost 2530000
+		"shields" 3000
+		"hull" 1500
+		"required crew" 1
+		"bunks" 5
+		"mass" 71
+		"drag" 1.25
+		"heat dissipation" 0.9
+		"fuel capacity" 800
+		"energy capacity" 300
+		"ramscoop" 0.5
+		"cargo space" 19
+		"outfit space" 108
+		"engine capacity" 54
+		"weapon capacity" 33
+		"shield generation" 1
+		"shield energy" 0.4
+		"hull repair rate" 2
+		"hull energy" 0.9
+		"cloak" .03
+		"cloaking energy" 3
+		"cloaking fuel" .05
+		"outfit scan power" 50
+		"outfit scan speed" 1
+		"tactical scan power" 50
+		"atmosphere scan" 100
+		"reverse thrust" 16
+		"reverse thrusting energy" 2.5
+		"reverse thrusting heat" 2.0
+		weapon
+			"blast radius" 20
+			"shield damage" 260
+			"hull damage" 160
+			"hit force" 800
+	outfits
+		"Millennium Cell"
+		"Emergency Ramscoop"
+		"Quantum Key Stone"
+		"Tuning Rifle" 1
```

Comment:
Yes, because I'm planning on boarding something with 5 crew total.
I don't see why this should be here stock.

Reply:
Heh. It isn't meant as a boarding weapon, it is meant as a hunting rifle. Or more accurately, a means of dealing with wildlife that has become threatening during the course of their investigations.